packages/ckeditor5-image/src/imageupload/imageuploadediting.js

```diff
@@ -192,6 +195,13 @@ export default class ImageUploadEditing extends Plugin {
 				}
 			}
 		} );
+
+		// Set the default handler for feeding the image element with `src` and `srcset` attributes.
+		this.on( 'uploadComplete', ( evt, { imageElement, writer, data } ) => {
```

Comment:
I think that we should not pass `writer` in the event params, we could here a separate change block - those are `'transparent'` because it's sth that happened "in the background" so it should not be undoable.

Reply:
Tell me if I'm wrong, but then the integrator would need to know to wrap the code in the `transparent` change block. With the writer from the `transparent` change block, he doesn't need to know about it. But I'm fine with both approaches.